config.app.js

```diff
@@ -48,7 +48,7 @@ function integerFromText(value, defaultValue) {
 const config = {
   VERSION: pkg.version,
   UPLOAD_API: __UPLOAD_API__ || 'https://tidepool.org/uploader',
-  API_HOST: __API_HOST__ || 'https://dev-api.tidepool.org',
+  API_HOST: __API_HOST__ || `${window.location.protocol}//${window.location.host}`.replace('app', 'api'),
```

Comment:
I would suggest changing the replacement from
.replace("app", "api") to
.replace("app.tidepool.org", "api.tidepool.org")
The reason is to avoid possible false positive matches for test environments where the name might be:
"\*app\*-app.tidepool.org".